scripts/services/rewrite-services-json.js

```diff
@@ -15,21 +15,29 @@ const { validateSvgIcons } = require('./validate-svg-icons');
  * @throws {Error} If there is an error while reading or parsing any of the YAML files, an error is thrown.
  */
 const getServiceFilesContent = async (dirPath, serviceFileNames) => {
-    try {
-        // Reads data from a yml file and writes it to an object
-        const serviceFileContent = serviceFileNames.map(async (fileName) => {
-            // Service file path
-            const serviceFilePath = path.resolve(__dirname, dirPath, `${fileName}${YML_FILE_EXTENSION}`);
-            // Read the file content
-            const fileChunk = await fs.readFile(serviceFilePath, 'utf-8');
-            const fileData = yaml.load(fileChunk);
-            return fileData;
-        });
+    const invalidYmlFiles = [];
+    // Reads data from a yml file and writes it to an object
+    const serviceFileContent = await Promise.all(
+        serviceFileNames.map(async (fileName) => {
+            try {
+                // Service file path
+                const serviceFilePath = path.resolve(__dirname, dirPath, `${fileName}${YML_FILE_EXTENSION}`);
+                // Read the file content
+                const fileChunk = await fs.readFile(serviceFilePath, 'utf-8');
+                const fileData = yaml.load(fileChunk);
+                return fileData;
+            } catch (error) {
+                // Collect the filename if an error occurred during reading
+                return invalidYmlFiles.push(fileName);
```

Comment:
`invalidYmlFiles.push(fileName);` will return a length of the invalidYmlFiles array after the push is executed. so this value should not be returned here
```suggestion
                invalidYmlFiles.push(fileName);
```